Resources/Prototypes/Reagents/toxins.yml

```diff
@@ -395,11 +407,12 @@
   color: "#D6CE7B"
   metabolisms:
     Poison:
+      metabolismRate: 0.90
```

Comment:
MetabolismRate is in units per second, with 0.5 being the base. This is almost twice as fast as normal which I assume you didn't intend.

Reply:
oops, meant to put 0.09